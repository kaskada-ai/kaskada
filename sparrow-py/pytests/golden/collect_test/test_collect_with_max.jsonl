{"_time":851013597000000000,"_subsort":0,"_key_hash":12960666915911099378,"_key":"A","m":5.0,"collect_m_max_2":[5],"n":10.0,"collect_n_max_2":[10]}
{"_time":851013598000000000,"_subsort":1,"_key_hash":2867199309159137213,"_key":"B","m":24.0,"collect_m_max_2":[24],"n":3.0,"collect_n_max_2":[3]}
{"_time":851013599000000000,"_subsort":2,"_key_hash":12960666915911099378,"_key":"A","m":17.0,"collect_m_max_2":[5,17],"n":6.0,"collect_n_max_2":[10,6]}
{"_time":851013600000000000,"_subsort":3,"_key_hash":12960666915911099378,"_key":"A","m":null,"collect_m_max_2":[5,17],"n":9.0,"collect_n_max_2":[6,9]}
{"_time":851013601000000000,"_subsort":4,"_key_hash":12960666915911099378,"_key":"A","m":12.0,"collect_m_max_2":[17,12],"n":null,"collect_n_max_2":[6,9]}
{"_time":851013602000000000,"_subsort":5,"_key_hash":12960666915911099378,"_key":"A","m":null,"collect_m_max_2":[17,12],"n":null,"collect_n_max_2":[6,9]}
