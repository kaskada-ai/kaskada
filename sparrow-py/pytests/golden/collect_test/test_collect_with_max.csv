_time,_subsort,_key_hash,_key,m,collect_m_max_2,n,collect_n_max_2
1996-12-20 00:39:57,0,12960666915911099378,A,5.0,[5.],10.0,[10.]
1996-12-20 00:39:58,1,2867199309159137213,B,24.0,[24.],3.0,[3.]
1996-12-20 00:39:59,2,12960666915911099378,A,17.0,[ 5. 17.],6.0,[10.  6.]
1996-12-20 00:40:00,3,12960666915911099378,A,,[17. nan],9.0,[6. 9.]
1996-12-20 00:40:01,4,12960666915911099378,A,12.0,[nan 12.],,[ 9. nan]
1996-12-20 00:40:02,5,12960666915911099378,A,,[12. nan],,[nan nan]
